<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_add_system</class>
 <widget class="QDialog" name="dialog_add_system">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#dialog_add_system
{
background-color:white;
}</string>
  </property>
  <widget class="QWidget" name="widget_top" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_top
{
border: 1px solid #000000;
background-color: rgb(157, 196, 229);
background-color: rgb(157, 196, 229);
}</string>
   </property>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>5</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/image/closeWindow.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>351</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget
{
border: 1px solid #000000;
}</string>
   </property>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_3
{
border: 1px solid #000000;
}

QLabel
{
font-family: '黑体';
font-size: 16px;
}

QComboBox
{
font-family: '黑体';
font-size: 16px;
}

QLineEdit
{
font-family: '黑体';
}

QRadioButton
{
font-family: '黑体';
}

QPushButton
{
font-family: '黑体';
}</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>设备类型</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_type">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="placeholderText">
          <string>请选择设备类型</string>
         </property>
         <item>
          <property name="text">
           <string>风机</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>风机盘管</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>空调器(单风机)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>空调器(双风机)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_num">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>编号</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_num">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="placeholderText">
          <string>请选择设备编号</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_model">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>型号</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_model">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_model">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>请选择设备型号</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_queding">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	color:#ffffff; /*文字颜色*/
    background-color: #00ACB9;/*背景色*/
    border-radius:0px; /*边框倒角*/
	border: 1px solid #000000;
}


QPushButton:hover 
{
background-color: rgb(0,142,153);
color: white;
}</string>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>11</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>141</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { 
border: none; 
color : rgb(105, 105, 105);
}</string>
    </property>
    <property name="text">
     <string>添加设备</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>11</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>41</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { 
border: none; 
background-color : rgb(105, 105, 105);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_model</tabstop>
  <tabstop>pushButton_queding</tabstop>
  <tabstop>comboBox_type</tabstop>
  <tabstop>comboBox_num</tabstop>
  <tabstop>close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
